array-manipulation-3/src/flatten.js

```diff
@@ -0,0 +1,14 @@
+/* exported flatten */
+/* result []
+   for subarray in array
+   push contents of subarray onto result
+*/
+function flatten(array) {
+  var result = [];
+  for (var i = 0; i < array.length; i++) {
+    for (var j = 0; j < array[i].length; j++) {
+      result.push(array[i][j]);
+    }
```

Comment:
Did this pass all tests? I don't imagine this would work for values that are already not arrays 🤔